test/PDPVerifier.t.sol

```diff
@@ -593,35 +609,45 @@ contract PDPVerifierProofTest is Test, ProofBuilderHelper {
     receive() external payable {}
 
     function testProveWithDifferentFeeAmounts() public {
+        // Mock Pyth oracle call to return $5 USD/FIL
+        bytes memory pythCallData = abi.encodeWithSelector(
+            IPyth.getPriceNoOlderThan.selector,
+            pdpVerifier.FIL_USD_PRICE_FEED_ID(),
+            86400
+        );
+        PythStructs.Price memory price = PythStructs.Price({
+            price: 5,
+            conf: 0,
+            expo: -10,
+            publishTime: 0
+        });
+        vm.mockCall(address(pdpVerifier.PYTH()), pythCallData, abi.encode(price));
+
         uint leafCount = 10;
         (uint256 setId, bytes32[][] memory tree) = makeProofSetWithOneRoot(leafCount);
 
         // Advance chain until challenge epoch.
         uint256 challengeEpoch = pdpVerifier.getNextChallengeEpoch(setId);
         vm.roll(challengeEpoch);
 
+        vm.mockCall(pdpVerifier.RANDOMNESS_PRECOMPILE(), abi.encode(challengeEpoch), abi.encode(challengeEpoch));
+
         // Build a proof with multiple challenges to single tree.
         uint challengeCount = 3;
         PDPVerifier.Proof[] memory proofs = buildProofsForSingleton(setId, challengeCount, tree, leafCount);
 
-        // Calculate the correct proof fee
-        uint256 correctProofFee = PDPFees.proofFee(challengeCount);
+        // Mock block.number to 2880
+        vm.roll(2880);
 
         // Test 1: Sending less than the required fee
-        uint256 insufficientFee = correctProofFee - 1;
+
+        uint256 correctFee = 388051072754688;
```

Comment:
Thanks, overall better to have this than not I think.  Please leave a comment stating why we need to to it this way.  If there is any more development on proving code we will run into failures every time here so at least we should make it easy to fix them.  Probably a good idea to leave instructions here for that too.

Reply:
Done.